react/features/toolbox/components/web/Toolbox.js

```diff
@@ -71,6 +70,7 @@ import {
     OverflowMenuVideoQualityItem,
     VideoQualityDialog
 } from '../../../video-quality';
+import { toggleSharedVideo } from '../../../youtube-player/actions';
```

Comment:
Please move the action to shared-video. Let's get rid off youtube-player altogether